include/vcpkg/base/messages.h

```diff
@@ -875,6 +875,7 @@ namespace vcpkg
     DECLARE_MESSAGE(CmdDependInfoOptDepth, (), "", "Show recursion depth in output");
     DECLARE_MESSAGE(CmdDependInfoOptDGML, (), "", "Creates graph on basis of dgml");
     DECLARE_MESSAGE(CmdDependInfoOptDot, (), "", "Creates graph on basis of dot");
+    DECLARE_MESSAGE(CmdDependInfoOptMermaid, (), "", "Creates graph on basis of mermaid");
```

Comment:
```suggestion
    DECLARE_MESSAGE(CmdDependInfoOptMermaid, (), "'mermaid' is a file format type that should remain unlocalized, see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams", "Creates graph on basis of mermaid");
```

Reply:
Also should we change "Creates graph on basis of BLAH" to "Create graph in BLAH format" ?